fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -30,13 +30,13 @@ class GridSearch(ReductionsEstimator):
 
     def __init__(self,
                  learner,
-                 disparity_metric,
+                 constraint,
```

Comment:
This worries me. We'll have more constraints in the future, right? So this should be plural.

Reply:
It should certainly be consistent with what `ExpGrad` ends up with. Although there, even if we combine different constraining moments, we should still end up with another moment object.